util/host.go

```diff
@@ -37,3 +41,18 @@ func EnsureHost(addr string) string {
 	}
 	return net.JoinHostPort(host, port)
 }
+
+// EnsureFullAddr check if addr is a host:port pair. It make sure to have both.
+func EnsureFullAddr(addr string) string {
```

Comment:
maybe the caller should pass in the default address so that we don't have to have that constant `26257` everywhere.